mobile/lib/constants/language_contants.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+const String LANGUAGE_CODE = 'languageCode';
+
+//languages code
+const String english = 'en';
+const String french = 'fr';
+const String portuguese = 'pt';
+const String swahili = 'sw';
+
+Future<Locale> setLocale(String languageCode) async {
```

Comment:
@katendejericho5 These can be removed since they're duplicates of what's in app service.